var/spack/repos/builtin/packages/gengetopt/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Gengetopt(AutotoolsPackage):
+    """Tool to write command line option parsing code for C programs"""
+
+    homepage = "https://www.gnu.org/software/gengetopt/gengetopt.html"
```

Comment:
Can you add a URL for the latest version here as well? `url_for_version` is used for fetching, but `url` is used for `spack versions` and `spack checksum`.

Reply:
Should I use the .xz or the .gz extension?

Reply:
I would add the newer one so you’ll be able to find new versions in the future.